cmd/kubelet/app/server.go

```diff
@@ -560,7 +560,7 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies) (err error) {
 		}
 
 		var clientCertificateManager certificate.Manager
-		if s.RotateCertificates && utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletClientCertificate) {
+		if s.KubeletConfiguration.RotateCertificates && utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletClientCertificate) {
```

Comment:
Both KubeletFlags and KubeletConfig are inline in KubeletServer, so you shouldn't have to qualify it when you move it.